.gitignore

```diff
@@ -0,0 +1,5 @@
+*.pyc
+data
```

Comment:
I'd remove `data` from the .gitignore as it's not created by the training script and the repo never specifies a default data location.